configs/mainnet.json

```diff
@@ -483,5 +531,119 @@
       "label_denom": "brdg/a02afc2c1edf77cc023eefa25fc036c184612faf9365cda9c1daa3b1675ebf8f",
       "target_denom": "brdg/a02afc2c1edf77cc023eefa25fc036c184612faf9365cda9c1daa3b1675ebf8f"
     }
-  ]
-}
+  ],
+  "chain_fee_token_map": {
+    "Archway": {
+      "denom": "ibc/652218B355FBD814037EEF28505F3D91F0489325A64FA5D865EF425DD59317BC",
+      "decimals": 18
+    },
+    "Axelar": {
+      "denom": "ibc/062588846168EDFECF20F7681F2A0A131A53B9A3C8535BCA8F7D1A268246974B",
+      "decimals": 6
+    },
+    "DYDX": {
+      "denom": "ibc/9DC57FF60E125F9B976EB0DF4F2122C580AD1DB8E27BA71E6D2C1AA56A5FD312",
+      "decimals": 18
+    },
+    "Osmosis": {
+      "denom": "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518",
+      "decimals": 6
+    },
+    "Cosmos Hub": {
+      "denom": "ibc/A4DB47A9D3CF9A068D454513891B526702455D3EF08FB9EB558C561F9DC2B701",
+      "decimals": 6
+    },
+    "Kujira": {
+      "denom": "ibc/662914D0C1CEBCB070C68F061D035E8B10A07C79AB286E7342C85F3BE74612C5",
+      "decimals": 6
+    },
+    "Noble": {
+      "denom": "usdc.1.6.53ff75",
```

Comment:
For Noble, the correct denom is `ibc/43897B9739BD63E3A08A88191999C632E052724AB96BD4C74AE31375C991F48D`.